tools/hlslpm/issue.py

```diff
@@ -27,7 +27,7 @@ def getReferenceFromTitle(self):
         if not self.title:
             return None
 
-        m = re.match(".*\((.+)\)", self.title)
+        m = re.match(r".*\((.*#\d+)\)", self.title)
```

Comment:
Something I tend to forget about is that they added assignment expressions in python 3.8 (https://docs.python.org/3/reference/simple_stmts.html#grammar-token-python-grammar-assignment_stmt).

Not sure it would be better here, but thought I'd throw it out in case it was new to you.
```
if m := re.match(...):
  # m is valid here
```

Reply:
Thanks, that was new to me.